test/e2e/specs/editor/plugins/annotations.spec.js

```diff
@@ -55,7 +55,7 @@ test.describe( 'Annotations', () => {
 		// There should be no <mark> tags in the raw content.
 		await expect(
 			block.locator( '.block-editor-block-list__block-html-textarea' )
-		).toHaveValue( '<p>Paragraph to annotate</p>' );
+		).toHaveText( '<p>Paragraph to annotate</p>' );
```

Comment:
Without this change, the test will fail with an `expect.toHaveValue: Error: Not an input element` error.